### PR TITLE
Add search bar #909

### PR DESCRIPTION
**Pull request by DeOldSax**
Added a simple search bar for looking up countries in the ranking list.

### CODE REVIEW

**Comment by corradio** [comment]
Quite cool @DeOldSax ! I'll take a look at this as soon as I have a chance.

**Comment by corradio** [comment]
Screenshot of mobile view:
image
Screenshot of desktop view:
image

File: web/locales/en.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
     },
     "rankings-panel": {
         "country-ranking": "Ranking",
-        "show-ranking": "Show ranking"
+        "show-ranking": "Show ranking",
+        "search": "Search..."
```
Comment (corradio) [inline web/locales/en.json]: Can you rename to `Search a zone`, without the trailing `.`?

File: web/app/main.js [hunk 2]
```diff
@@ -1111,6 +1111,24 @@ function dataLoaded(err, clientVersion, argCallerLocation, state, argSolar, argW
         countryTooltip.hide();
     });
 
+    // Add search bar handler
+    var selector = d3.select('.country-search-bar input')
```
Comment (corradio) [inline web/app/main.js]: Would be nice to write ES6 code instead (we're slowly moving everything to ES6).
It's ok if you can't do it right now.

**Comment by corradio** [comment]
Looks very nice. @jarek @brunolajoie @systemcatch any comments?

**Comment by DeOldSax** [comment]
@corradio thx for the review - let me know if you want me clean this up to have a single commit

**Comment by corradio** [comment]
@DeOldSax all is good - just making sure we're all on the same page regarding the issue.
I'll squash commits into one when merging.
Can you however merge `master` into your branch?

**Comment by brunolajoie** [comment]
Nie feature! But when I test it locally on chrome it does not looks similar to @corradio screenshot and the search does not work when pressing enter. 
image

**Comment by corradio** [comment]
@brunolajoie there's probably something wrong with your setup.
Can you try building from master? Also, check the console, maybe an error is hidden there.

Olivier

**Comment by jarek** [comment]
Works for me on the prod site now. Maybe it was a cache problem for Bruno?

**Pull request merged**